tools/qemu/Dockerfile

```diff
@@ -1,15 +1,20 @@
-FROM alpine:edge
+FROM linuxkit/alpine:5240cbd9cf371c8211c8f1968e57c51a32098c8f AS mirror
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add --no-cache --initdb -p /out \
+    alpine-baselayout \
+    busybox \
+    libarchive-tools \
+    qemu-img \
+    qemu-system-arm \
```

Comment:
I don't know that we need `qemu-system-arm` in here now - we are not using it anywhere now.